ia/src/ml_agent.py

```
@@ -121,6 +122,34 @@
 
   return analyzed_vision
 
+def strategy(client):
```

Ruff (0.11.9)

Missing return type annotation for public function `strategy`

Add return type annotation: `None`

(ANN201)